schemas/io.snowplow.ecomm/click_item/jsonschema/1-0-0

```diff
@@ -0,0 +1,36 @@
+{
+    "description": "Event occurrs when an item (a product or bundle) is clicked on.",
+    "properties": {
+        "item_id": {
+            "type": "string",
+            "description": "ID reference to the item (as listed in item) that was clicked on.",
+            "maxLength": 255
+        },
+        "item_type": {
+            "type": "string",
```

Comment:
Should this. be a required property or should we allow it to be null?

Reply:
So after discussing with the other guys we agree that the item type field should be required.
It is also a join key used in our data models so to ensure that users will use the correct values lest we break any downstream models we've also opted for making it an enum field.

Reply:
That makes sense! I just meant that if it is required then we should have it in the list of `required` properties (currently there is only `item_id`)? Otherwise it's in a weird state where we don't accept null for it but it can be missing from payload.